### DISCUSSION TITLE
An intermediate release before 1.1

### DISCUSSION THREAD

**Discussion by ihhub** [ihhub/fheroes2#8267] [General]
Based on the progress on the Editor I have doubts that we can make 1.1 release in the specified time. I am suggesting to create an intermediate release and delay 1.1. This will be a compromise between waiting time for players and their expectations. We would make another release in a week from now.

I would like to hear your opinion, @oleg-derevenetz 🙂

**Comment by oleg-derevenetz** [comment]
Hi @ihhub we can also postpone the 1.1, but leave the release date of the next release unchanged (i.e. February 7). Are there any new features that you definitely want to roll out to the public during the upcoming week?

**Reply by ihhub** [reply]
This makes even more sense. I created 1.0.12 milestone for 7th February. I think we shouldn't rush too much and prepare everything for the release. 1.0 was shifted to 7th March.
